Lib/test/test_frame.py

```diff
@@ -420,6 +421,17 @@ def test_unsupport(self):
         with self.assertRaises(TypeError):
             copy.deepcopy(d)
 
+    def test_is_mapping(self):
+        x = 1
```

Comment:
Why do you need `x`?

Reply:
The contents are irrelevant, but we want something just to make sure we are testing the right thing.